news/2018-12-31-beatmap-spotlights-fall-2018.md

```diff
@@ -0,0 +1,250 @@
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/envidia.jpg)](http://osu.ppy.sh/beatmapsets/819153)
+
+The autumn season of mania ranked seemed to mark a heavy interest in the long note side of things, and [Calcyon](https://osu.ppy.sh/users/2412652)'s addition in Nekomata Gekidan's "envidia" is certainly up there with the best of them this season. This marathon makes extremely nice use of varying LN patterns to mark key differences in sections of the map, using lighter layering, trills, inverse, and even some minijacks to clearly define the intensity of different sections. 
+
+All of this is paired with difficult yet smooth transitions and patterning to make the experience all the more fun and rewarding for players. Envidia, through these choices, certainly proved itself to be a point of quality for the autumn season of mania.
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/smiling.jpg)](http://osu.ppy.sh/beatmapsets/810911)
+
+[Kawawa](https://osu.ppy.sh/users/4647754) comes back with a complete shift of his mapping style along with Lime's "Smiling". Kawawa interprets the melody line with complex long notes with its flow, mixed with harsh chord stream in between. As the song progresses, players will have to  go through fast streams and moreover read out the full long note chorus section. It is indeed a challenging map but also enjoyable for long note lovers. 
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/genesis.jpg)](http://osu.ppy.sh/beatmapsets/779906)
+
+Hakuryu's "Genesis At Oasis (MOONLiGHT Mix)" mapped by [Alternater](https://osu.ppy.sh/users/4980256) featured in 4K MWC 2018, the top difficulty has already been well-known before it was made into this great full spread. Even though the patterns used contains mostly variations of basic jumpstreams, they closely fit with the music and therefore well represent the whole song. The SV use is also very comfortable and beneficially exaggerates the musical impact. 
+
+[-MysticEyes](https://osu.ppy.sh/users/6253266), [Unpredictable](https://osu.ppy.sh/users/7560872), and [qqqant](https://osu.ppy.sh/users/8976576) also thoughtfully contribute to the set by mapping solid lower difficulties for this set. Overall, this set expresses the music perfectly and deserves a place in Spotlight!
+
+[![](/wiki/shared/news/2018-12-31-beatmap-spotlights-fall-2018/mania/dream-catcher.jpg)](http://osu.ppy.sh/beatmapsets/696380)
+
+Here [Salty Mermaid](https://osu.ppy.sh/users/9037365) makes his debut into the mapping scene with his song, MYLK's "Dream Catcher". The most notable and memorable of this map is by far the kiai with its fast paced patterns implementing both jacks and trills in a way that heavily compliment the song and makes the song that much more enjoyable upon revisiting.
```

Comment:
heavily compliment the song and make* the song although it's redundant, remove the first "the song"